Dockerfile

```diff
@@ -31,13 +28,17 @@ COPY environment.conf.${BUILD_ENV}.json /app/environment.conf.json
 RUN git config --global url."https://".insteadOf git://
 RUN git config --global url."https://github.com/".insteadOf git@github.com:
 
-
 # Install NPM dependencies. Also:
 RUN npm --production=false \
         --unsafe-perm \
         --verbose \
-       install
-RUN npm cache clean --force
+       install && npm cache clean --force
+
+# Copy sources
+COPY src /app/src
```

Comment:
I would move this line below line 40; code in `src` is more likely to change than code in the `internals` and `server` folders

Reply:
:)